src/domain/eventSearch/searchResultList/__tests__/ResultsInfo.test.tsx

```diff
@@ -131,3 +134,46 @@ test.each([1, 4])(
     });
   }
 );
+
+it.each([
+  [4, 'en', 'event', `/fi${ROUTES.EVENTS}`],
+  [0, 'en', 'event', `/fi${ROUTES.EVENTS}`],
+  [4, 'en', 'course', `/fi${ROUTES.COURSES}`],
+  [0, 'en', 'course', `/fi${ROUTES.COURSES}`],
+  [0, 'sv', 'event', `/fi${ROUTES.EVENTS}`],
+  [0, 'sv', 'course', `/fi${ROUTES.COURSES}`],
+])(
+  'renders language change button under search results when current language is %s and there are %i %s search items',
```

Comment:
This is logging NaN:

`renders language change button under search results when current language is 4 and there are NaN event search items`